debian/README.source

```diff
@@ -0,0 +1,29 @@
+This package is maintained with git-buildpackage(1). It follows DEP-14 for
```

Comment:
I don't see why this file is necessary.  Just drop it.  It's all Debian-specific comments.